server/src/main/java/org/elasticsearch/repositories/blobstore/BlobStoreRepository.java

```diff
@@ -2089,6 +2089,7 @@ private void executeOneFileRestore(BlockingQueue<BlobStoreIndexShardSnapshot.Fil
 
                 private void restoreFile(BlobStoreIndexShardSnapshot.FileInfo fileInfo, Store store) throws IOException {
                     ensureNotClosing(store);
+                    logger.trace(() -> new ParameterizedMessage("[{}] Restoring [{}] to [{}]", metadata.name(), fileInfo, store));
```

Comment:
It's far from being uniform in the codebase but I think that log messages are usually not capitalized whereas exception messages are.

`R`-> `r`